File: HFS-BE/HFS-BE/Models/Category.cs
using System;
using System.Collections.Generic;

namespace HFS_BE.Models
{
    public partial class Category
    {
        public Category()
        {
            Foods = new HashSet<Food>();
        }

        public int CategoryId { get; set; }
        public string? Name { get; set; }
        public bool? Status { get; set; }

        public virtual ICollection<Food> Foods { get; set; }
    }
}

File: HFS-FE/src/app/modules/admin-routing-module/models/Category.ts
export class Category {
    categoryId :number ;
    name :string ;
    status :string ;
  }

  export class CategoryInput {
    categoryId :number ;
    name :string ;
    status :string ;
  }

  export class CateDisplayHideInputDto{
    categoryId: number;
    type: boolean;
  }